src/manager.cpp

```diff
@@ -1,7 +1,11 @@
+#include "config.h"
+
 #include "manager.hpp"
 
 #include "logger.hpp"
 
+#include <boost/asio/steady_timer.hpp>
+
```

Comment:
we decided to not have short names which is confusing .
line 15: m_conn(conn) <-- we could have a name  saying what  is this connection for.